chart/templates/configmap-env.yaml

```diff
@@ -12,7 +12,7 @@ data:
   {{- end }}
   DB_NAME: {{ .Values.postgresql.auth.database }}
   DB_POOL: {{ .Values.mastodon.sidekiq.concurrency | quote }}
-  DB_PORT: "5432"
+  DB_PORT: {{ .Values.postgresql.auth.port | default "5432" | quote }}
```

Comment:
There's a separate PR tackling this specifically so it might be a good idea to drop it from here
(and I guess technically the port shouldn't be in `auth`. database is in there because it's sort of "related" to the authentication being used)
I Think port should probably be on the same level as the external hostname which is slightly confusing, but the place bitnami has it is bizarre and you'll probably only change this if you're also specifying the hostname

Reply:
Do you have a link to the other PR @deepy ? I was going to raise a PR to set a non-standard Postgres port too, would be great to track it.

Reply:
https://github.com/mastodon/mastodon/pull/20370